.github/workflows/nightly.yml

```
@@ -0,0 +1,52 @@
+        env:
+          GH_TOKEN: ${{ github.token }}
+          SHA: ${{ github.event.workflow_run.head_sha }}
+        uses: softprops/action-gh-release@v3
+        with:
+          tag_name: nightly
+          name: Nightly
+          prerelease: true
+          make_latest: false
+          target_commitish: ${{ github.sha }}
+          files: ${{ env.AssetName }}
+          body: "Automated nightly build from \`main\` @ ${{ github.sha }}."
```

actionlint (1.7.12)

[error] could not parse as YAML: found unknown escape character

(syntax-check)